hw4/phonebook/phonebook.c

```diff
@@ -0,0 +1,168 @@
+﻿#pragma warning (disable : 6031)
+
+#include <stdio.h>
+#include <time.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define ARRAY_SIZE 100
+#define RANGE 11
+#define _CRT_DEFINE_NO_WARNINGS
```

Comment:
```suggestion
#define _CRT_SECURE_NO_WARNINGS
```
И её надо писать до библиотечных заголовочных файлов, желательно в самом начале.  Тут поздно уже :)